spec/src/main/doc/core/implementation.asciidoc

```diff
@@ -730,7 +494,7 @@ A bean constructor may have any number of parameters. All parameters of a bean c
 
 === Injected fields
 
-An _injected field_ is a non-static, non-final field of a bean class or of any Java EE component class supporting injection.
+An _injected field_ is a non-static, non-final field of a bean class.
```

Comment:
The expression "Java EE component class supporting injection" is not treated consistently in this PR. Here, the expression is eliminated whereas in `injectionandresolution.asciidoc` for example this expression is replaced by "other classes supporting injection". The expression should be dealt with consistently across the entire document.